mobile/build.gradle

```diff
@@ -37,10 +37,10 @@ repositories {
 android {
 
     compileSdkVersion 24
-    buildToolsVersion "24.0.0"
+    buildToolsVersion '25.0.0'
     defaultConfig {
         applicationId "mycroft.ai"
-        minSdkVersion 21
+        minSdkVersion 23
```

Comment:
I don't think so, but as I said to seymour, I will look into it. I remember when writing the code that it complained about this. Otherwise I tend not to change the min if I can avoid it.